addons/web/static/src/core/colorlist/colorlist.xml

```diff
@@ -2,12 +2,12 @@
 <templates xml:space="preserve">
 
     <t t-name="web.ColorList" owl="1">
-        <div class="o_colorlist" aria-atomic="true" t-ref="colorlist">
-            <t t-if="props.togglerColor !== undefined and !state.isExpanded">
-                <button t-on-click.prevent.stop="onToggle" role="menuitem" t-att-title="colors[props.togglerColor]" t-att-data-color="props.togglerColor" t-att-aria-label="colors[props.togglerColor]" t-attf-class="{{'btn o_colorlist_toggler o_colorlist_item_color_' + props.togglerColor }}"/>
+        <div class="o_colorlist d-flex flex-wrap align-items-center mw-100 gap-2" aria-atomic="true" t-ref="colorlist">
+            <t t-if="!props.isExpanded and !state.isExpanded">
```

Comment:
I don't like this. state should be updated when the props change, not check both of them independently

Reply:
and so if you have state.something, props.something should never be used aside update the state.something